env.example

```diff
@@ -62,6 +62,9 @@ IRC_QUIT_MESSAGE="TeleIRC bridge stopped."
 TELEGRAM_CHAT_ID=-0000000000000
 TELEIRC_TOKEN=000000000:AAAAAAaAAa2AaAAaoAAAA-a_aaAAaAaaaAA
 MAX_MESSAGES_PER_MINUTE=20
+TELEGRAM_MESSAGE_REPLY_PREFIX="["
+TELEGRAM_MESSAGE_REPLY_SUFFIX="]"
```

Comment:
This is fine here because it's consistent with how everything already works, but as a tangential thought I wonder if configuration-wise it would be easier to allow specifying a format string instead of prefix and suffix for things.

Reply:
What exactly do you mean by format string?

Reply:
I would guess something like `[re: %s]` where `%s` is the shortened reply substring.

Reply:
Exactly, a printf-style format string